src/search/evaluators/g_evaluator.cc

```diff
@@ -8,6 +8,10 @@
 using namespace std;
 
 namespace g_evaluator {
+GEvaluator::GEvaluator(const options::Options &opts)
```

Comment:
I have never heard of this. Do you know of any use in Fast Downward?

Reply:
Much of our code is older than C++11, and also I'm not sure how often the scenario arises in our code where we have a nontrivial constructor that we would like to inherit, and nothing "new" needs to be done in the derived class constructor. I guess usually there would be something in the derived class that we need to take care of in the constructor.

It could also be the case that we want to inherit some, but not all constructors for some reason, and in that case I don't think `using` is an option. Maybe also some people have a preference for the more explicit form so that they can more easily see how an object is constructed. In any case, I'm not aware of any objections that someone has raised against inheriting constructors.